.prettierignore

```diff
@@ -1 +1,2 @@
 package.json
+/codemods/
```

Comment:
Any reason we _don't_ want to have prettier run on the codemods dir?

Reply:
Yes – well mainly the test fixtures – I couldn't get the test runner to apply our prettier/babel config so the output matched properly. Any tips here?